.perltidyrc

```diff
@@ -2,11 +2,9 @@
 # # See <https://github.com/houseabsolute/perl-code-tidyall/issues/84>.
 --character-encoding=none
 --no-valign
-# 120 characters would be desired but it is not feasible right now
-#-l=120   # 120 characters per line
```

Comment:
We have 160 in the test distribution, right? I don't think we should try to ensure the same level of quality into both low level software as well as the test distribution so I would keep the stricter line length. Maybe it's possible to override settings with multiple config files?

Reply:
I disagree, 120 or 160 isn't really the end of the world, and I would much rather keep the same standard for everybody, also there is test code (albeit baseclass and company) in os-autoinst at least.

Reply:
> I disagree, 120 or 160 isn't really the end of the world, and I would much rather keep the same standard

I second that. An extra config file for a tiny difference doesn't really warrant the added overhead.

Reply:
@okurz there's no support for extra profile files sadly. See the link below:

https://github.com/perltidy/perltidy/blob/master/lib/Perl/Tidy.pm#L4218-L4223

In any case this is just aligning the contents with the PR in `os-autoinst-commons` (removing two comments and adding a space). The config is exactly the same.

Reply:
@okurz I guess you still have the objection but no further discussion is needed, right?

Reply:
Yeah fine by me. Please update the git commit message, not just "Experiment"